composer.json

```diff
@@ -9,7 +16,7 @@
 
 	},
 	"require-dev": {
-		"christophwurst/nextcloud": "dev-master@dev",
+		"christophwurst/nextcloud_testing": "^0.12.4",
```

Comment:
What is "christophwurst/nextcloud_testing" ?

Reply:
https://github.com/ChristophWurst/nextcloud_testing

Reply:
But so is it correct to remove "christophwurst/nextcloud" ? It is not the same thing "christophwurst/nextcloud" contains the OCP stubs for static analysis no?